app/models/channel.js

```diff
@@ -15,4 +15,16 @@ export default Ember.Object.extend({
     return this.get('name').replace(/#/g,'');
   }.property('name'),
 
+  formattedTopic: function() {
+    if (this.get('topic') !== null) {
+      let topic = linkifyStr(this.get('topic'), {
+        defaultProtocol: 'https',
+        linkAttributes: { rel: 'nofollow' }
+      });
+      return new Ember.String.htmlSafe(topic);
```

Comment:
Not sure this is a good idea. Marking the whole `topic` as `htmlSafe` could allow XSS.